pkgs/racket-doc/scribblings/reference/sequences.scrbl

```diff
@@ -1152,15 +1152,19 @@ stream, but plain lists can be used as streams, and functions such as
 
  @history[#:added "8.0.0.12"]}
 
-@defform[(stream expr ...)]{
+@defform[(stream e ...)]{
```

Comment:
Why is `expr` changed to `e`? `expr` follows the [Notation for Syntactic Form](https://docs.racket-lang.org/reference/notation.html#%28part._.Notation_for_.Syntactic_.Form_.Documentation%29), so I think it's better to keep the original name.

Reply:
It might make sense to keep as `expr`, but I changed it because now those can also be patterns too so I thought `expr` might be misleading.

Reply:
Typically the documentation for constructors doesn't worry about this; for example `list` or `cons`.

Reply:
A `deftogether` with one case for constructor and another case for pattern could also work, IMO.